org.lflang/src/org/lflang/generator/c/CGenerator.java

```diff
@@ -421,6 +421,9 @@ public void accommodatePhysicalActionsIfPresent() {
                             "Using the threaded C runtime to allow for asynchronous handling of physical action " +
                             action.getName()
                         );
+                        if (targetConfig.fastMode) {
```

Comment:
This doesn't look quite right. This check is performed only when threading is turned off. Shouldn't it always be performed?